[2023-03-01 13:08:09,427] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-12-06T19:05:00+00:00 [queued]>
[2023-03-01 13:08:09,699] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-12-06T19:05:00+00:00 [queued]>
[2023-03-01 13:08:09,706] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 13:08:09,707] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2023-03-01 13:08:09,709] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 13:08:09,930] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): get_name> on 2022-12-06T19:05:00+00:00
[2023-03-01 13:08:09,969] {standard_task_runner.py:52} INFO - Started process 19933 to run task
[2023-03-01 13:08:10,020] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'with_python_operator', 'get_name', '2022-12-06T19:05:00+00:00', '--job-id', '4771', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/02_python_operator.py', '--cfg-path', '/tmp/tmpk_5l39bz', '--error-file', '/tmp/tmplfdwl2ac']
[2023-03-01 13:08:10,028] {standard_task_runner.py:77} INFO - Job 4771: Subtask get_name
[2023-03-01 13:08:10,495] {logging_mixin.py:104} INFO - Running <TaskInstance: with_python_operator.get_name 2022-12-06T19:05:00+00:00 [running]> on host abe4c4997ef7
[2023-03-01 13:08:11,147] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dataheroes
AIRFLOW_CTX_DAG_ID=with_python_operator
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2022-12-06T19:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-06T19:05:00+00:00
[2023-03-01 13:08:11,340] {python.py:118} INFO - Done. Returned value was: None
[2023-03-01 13:08:11,539] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=with_python_operator, task_id=get_name, execution_date=20221206T190500, start_date=20230301T130809, end_date=20230301T130811
[2023-03-01 13:08:12,307] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-03-01 13:08:12,466] {local_task_job.py:146} INFO - Task exited with return code 0
